bucket/registryfinder.json

```diff
@@ -0,0 +1,55 @@
+{
+    "version": "2.52.0.0",
+    "description": "Registry Finder is an improved replacement for the built-in Windows registry editor",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://registry-finder.com/License.txt"
+    },
+    "homepage": "https://registry-finder.com/",
+    "notes": "Run \"Registry Finder\" from the Start Menu, or \"registryfinder.exe\" from the command line.",
```

Comment:
This is not required. The created shim and shortcut are the only way to launch (in this case).